openedx_events/learning/data.py

```diff
@@ -296,3 +296,22 @@ class ProgramCertificateData:
     status = attr.ib(type=str)
     url = attr.ib(type=str)
     certificate_available_date = attr.ib(type=datetime, default=None)
+
+
+@attr.s(frozen=True)
+class ExamAttemptData:
+    """
+    Attributes defined for the Open edX Exam downstream effects.
+
+    Arguments:
+        student_user (UserData): User object for the student to whom the exam attempt belongs
+        course_key (CourseKey): identifier of the course to which the exam attempt belongs
+        usage_key (UsageKey): identifier of the content that will get a exam attempt
+        requesting_user (UserData): user triggering the event (sometimes a non-learner, e.g. an instructor)
+        credit_requirement_status (str): what to change the student's CreditRequirement status to
```

Comment:
Docstring vs implementation for credit_requirement_status?

Reply:
Whoops! Fixing that now.

Reply:
For context, I removed that parameter and just forgot to remove the docstring.